server/tserver/src/main/java/org/apache/accumulo/tserver/ScanServer.java

```diff
@@ -408,6 +408,7 @@ public void run() {
     metricsInfo.addCommonTags(List.of(Tag.of("resource.group", groupName)));
 
     scanMetrics = new TabletServerScanMetrics();
+    sessionManager.setZombieCountConsumer(scanMetrics::setZombieScanThreads);
```

Comment:
I'm confused by this line. `TabletServerScanMetrics.setZombieScanThreads` returns `void`.

Reply:
You missed the `::`.  Its not calling the `setZombieScanThreads` on this line,  its passing a method reference.